src/rgw/rgw_rest_sts.cc

```diff
@@ -188,20 +189,20 @@ WebTokenEngine::get_from_jwt(const DoutPrefixProvider* dpp, const std::string& t
     if (error == -EACCES) {
       throw -EACCES;
     }
-    ldpp_dout(dpp, 5) << "Invalid JWT token" << dendl;
+    ldpp_dout(dpp, 5) << "Invalid JWT" << dendl;
     return boost::none;
   }
   catch (...) {
-    ldpp_dout(dpp, 5) << "Invalid JWT token" << dendl;
+    ldpp_dout(dpp, 5) << "Invalid JWT" << dendl;
```

Comment:
This debug statement looks exactly like the one on line 192.  It would almost certainly be helpful to capture the exception and stringify it out in the message.